### DISCUSSION TITLE
Has openobserve reached General Availability (GA)?

### DISCUSSION THREAD

**Discussion by cold01water** [openobserve/openobserve#1000] [Q&A]
The documentation still mentions an alpha version, 
but based on blog posts and cloud service announcements, 
it seems like it has transitioned to GA. What is the actual status?

**Comment by prabhatsharma** [comment]
Logs has been in production usage for quite a while and has hundreds of production deployments and should be considered GA.
Traces is closer to GA.
Metrics still has some areas that need work (PromQL, Ingestion etc) and should be considered alpha.

**Comment by cold01water** [comment]
Thanks for the honest answer.
